### PR TITLE
Contact us plz

### LINKED ISSUE #1307: action page (contact us) - Contact us modal needs to be styled
- [ ] Modal header should read "CONTACT US"
- [ ] "Enter your question..." should be in our body copy.
- [ ] Submit button should be our primary call to action button style

image

### PR DESCRIPTION
**Pull request by DFurnes**
Fixes #1307.

screen shot 2014-03-17 at 4 30 24 pm

### CODE REVIEW

**Comment by aaronschachter** [comment]
Now that's what I call a modal :musical_keyboard: :+1: much value added

File: lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module [hunk 1]
```diff
@@ -57,15 +57,21 @@ function dosomething_zendesk_get_client() {
  * Form constructor for submitting a Zendesk ticket.
  */
 function dosomething_zendesk_form($form, &$form_state)  {
+  dpm($form);
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module]: AHH :bomb:

Comment (DFurnes) [inline lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module]: HAHAHA GOD I SUCK

Comment (angaither) [inline lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module]: it's cause you didn't have :cookie:

**Comment by aaronschachter** [comment]
I take back my comment

**Comment by angaither** [comment]
:sweat_drops: :sweat:

**Comment by DFurnes** [comment]
:frowning:

**Pull request merged**